src/app/app.component.ts

```diff
@@ -171,6 +175,14 @@ export class AppComponent {
       this.footerDefaults = this.appConfig.APP_FOOTER_DEFAULTS;
       this.appAuthenticationDefaults = this.appConfig.APP_AUTHENTICATION_DEFAULTS;
       this.appFields = this.appConfig.APP_FIELDS;
+      this.languagesMeta = this.appConfig.APP_LANGUAGES_META;
+    });
+  }
+
+  /** Update meta for current language */
+  private subscribeToLanguageChanges() {
```

Comment:
nit(non-blocking) - it might be a little bit tidier to simply keep a `rtl` public property in the language service and bind that in the component html instead of having intermediate variable within the component ts. You could also avoid the need to use subscriptions by defining the variable using signals, so that the translate service sets the signal value on language change and the app component html reads the signal value